code/SiteConfig.php

```diff
@@ -168,6 +188,10 @@ public function getCMSFields()
             'SilverStripe\\CMS\\Model\\SiteTree.ACCESSONLYTHESE',
             "Only these groups (choose from list)"
         );
+        $viewersOptionsSource[InheritedPermissions::ONLY_THESE_MEMBERS] = _t(
+            self::class . '.ACCESSONLYTHESEMEMBERS',
+            "Only these users (choose from list)"
+        );
```

Comment:
I think in this block some mistake.
Placeholder says: **"Only these groups (choose from list)"**, but value for this radio button is **"OnlyTheseUsers"**.
For next radio button **"Only these users (choose from list)"** -> value = **"OnlyTheseMembers"**.
In my opinion it should be vice versa?!

It looks like an old mistake and it currently works as expected, but meaning is misleading.

Reply:
`OnlyTheseUsers` is the existing constant on `InheritedPermissions` and the existing value in the database. We can't change that without introducing a breaking change. So we have to leave that as it is, and introduce something new (`OnlyTheseMembers`) to track users.

Note that the decision to do that was done in https://github.com/silverstripe/silverstripe-framework/issues/10817 and already applies to pages and files - this PR just carries it through to siteconfig.